docs/sync/azure_pipelines.md

```diff
@@ -0,0 +1,316 @@
+[access_token]: https://learn.microsoft.com/azure/devops/pipelines/build/variables#systemaccesstoken
+[auth_scope]: https://learn.microsoft.com/azure/devops/pipelines/process/access-tokens#job-authorization-scope
+[PAT]: https://learn.microsoft.com/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+[phylum_tokens]: https://docs.phylum.io/docs/api-keys
+[phylum_contact]: https://phylum.io/contact-us/
+[phylum_register]: https://docs.phylum.io/docs/phylum_auth_register
+[phylum_project]: https://docs.phylum.io/docs/phylum_project
+[phylum_project_create]: https://docs.phylum.io/docs/phylum_project_create
+
+## Configure `azure-pipelines.yml`
+
+Phylum analysis of dependencies can be added to existing pipelines or on it's own with this minimal configuration:
+
+```yaml
+trigger:
+  - main
+
+jobs:
+  - job: Phylum
+    pool:
+      vmImage: ubuntu-latest
+    container: phylumio/phylum-ci:latest
+    steps:
+      - script: phylum-ci
+        displayName: Analyze dependencies with Phylum
+        env:
+          PHYLUM_API_KEY: $(PHYLUM_TOKEN)
+          AZURE_TOKEN: $(AZURE_PAT)
+```
+
+This single stage pipeline configuration contains a single container job named `Phylum`, triggered to run on pushes
+or PRs targeting the `main` branch. It does not override any of the `phylum-ci` arguments, which are all either
+optional or default to secure values.
+
+Let's take a deeper dive into each part of the configuration:
+
+### Pipeline control
+
+Choose when to run the pipeline. See the [YAML schema trigger definition][yaml_trigger] documentation for more detail.
+
+```yaml
+# This is a CI trigger that will cause the
+# pipeline to run on pushes to the `main` branch
+trigger:
+  - main
+```
+
+It is recommended to also enable PR validation for the target trigger branch(es). To do so, navigate to the branch
+policies for the desired branch (`main` in this example), and configure the [Build validation policy][build_validation]
+for that branch. For more information, see the documentation on [PR triggers for Azure Repos Git][pr_triggers] hosted
+repositories or, more broadly, [events that trigger pipelines][triggers].
+
+[yaml_trigger]: https://learn.microsoft.com/azure/devops/pipelines/yaml-schema/trigger
+[build_validation]: https://learn.microsoft.com/azure/devops/repos/git/branch-policies#build-validation
+[pr_triggers]: https://learn.microsoft.com/azure/devops/pipelines/repos/azure-repos-git#pr-triggers
+[triggers]: https://learn.microsoft.com/azure/devops/pipelines/build/triggers
+
+### Job names
+
+The job name can be named differently or included in an existing stage/job.
+
+```yaml
+jobs:
+  - job: Phylum  # Name this what you like
+```
+
+### Pool selection
+
+The pool is specified at the job level here because this is a [container job][container_job] and currently only
+`windows-2019` and `ubuntu-*` images support running containers. Keeping that restriction in mind, the pool can
```

Comment:
Yes...good point. There is not currently a `phylumio/phylum-ci` Docker Hub image available for Windows. The intent with this language was simply to repeat the limitations of Azure Pipelines when running container jobs, but that subtlety may be lost here. This will be updated to be more clear.